jina/parsers/orchestrate/runtimes/remote.py

```diff
@@ -10,17 +10,35 @@ def mixin_remote_runtime_parser(parser):
     :param parser: the parser
     """
     gp = add_arg_group(parser, title='RemoteRuntime')
-    _add_host(gp)
+    
+    gp.add_argument(
+        '--host',
+        '--host-in',
+        # type=str,
+        nargs = '+',
+        default=__default_host__,
+        action=CastHostAction,
+        help=f'The host of the Gateway, which the client should connect to, by default it is {__default_host__}.'
+        ' In the case of an external Executor (`--external` or `external=True`) this can be a list of hosts, separated by commas.'
```

Comment:
It can be a list of `hosts`, no need to say separated by commas. Or you have to say:

It can be: a list of hosts, a string with the hosts separated by commas or a single host